### DISCUSSION TITLE
Caiman background absorbs neurons

### DISCUSSION THREAD

**Discussion by LeoJuh** [flatironinstitute/CaImAn#1259] [Q&A]
Even though I am running my CaImAn analysis with one global background component, when I plot my background component it looks identical to the mean of my t-series. So the background is clearly absorbing signals from the neurons, what could be the issue here?

![caiman](https://github.com/flatironinstitute/CaImAn/assets/89864831/d8aec23b-b1c2-434c-97e9-a01237ff810c)

**Comment by EricThomson** [comment]
It is pretty common that the background look like the mean: indeed one common preprocessing step when SNR is low is to just subtract the mean to better bring out the signals so movies look more appealing.  Hence, if bg resembles the mean, I wouldn't worry much about that as a first pass. If the source extraction algorithm is missing *local fluctuations* (in time) in the cells' activity, then that would concern me and would suggest some parameters are off.
